src/Common/Core/Impl/Microsoft.Common.Core.csproj

```diff
@@ -88,7 +88,13 @@
     <Compile Include="Services\ICoreServices.cs" />
     <Compile Include="Services\CoreServices.cs" />
     <Compile Include="Shell\CoreShellExtensions.cs" />
+    <Compile Include="Shell\ExportImageParameters.cs" />
```

Comment:
Core assembly is generally for non-UI components and commonly used code. It does have some, but we are in process of cleaning this up post 1.0. Please move everything that is this dialog-related to Package/Plots.